test/Piece.spec.js

```diff
@@ -0,0 +1,82 @@
+QUnit.test("Piece constructor makes an empty piece", function (assert) {
+	const piece = new Piece();
+	assert.deepEqual(piece.moveVectors, []);
+	assert.deepEqual(piece.captureVectors, []);
+	assert.deepEqual(piece.moveCaptureVectors, []);
+	assert.deepEqual(piece.identifier, "");
+	assert.deepEqual(piece.player, undefined);
+});
+
+QUnit.test("Piece.setMoveVectors correctly sets moveVectors", function (assert) {
+	const piece = new Piece();
+	assert.deepEqual(piece.moveVectors, []);
```

Comment:
These initial asserts on `moveVectors`, `captureVectors`, and `moveCaptureVectors` are unnecessary. You already test them in the first test. Moreover, we get consistency with the arrange, act, assert methodology for what that's worth.